src/stories/radioButton.stories.tsx

```diff
@@ -28,7 +28,12 @@ export default {
   },
 }
 
-const Template = (args) => {
+interface Props extends RadioButtonProps {
+  storyName: string
+  label: JSX.Element
```

Comment:
we keep repeating the definition for `storyName` and an optional `label` in every story we write. if we defined a type for it, we could enforce some common structure to our stories. wdyt? worth it?

Reply:
I'll let it up to you to decide. The conversion to tsx is worth so much on its own. I could totally live with some duplication here

Reply:
Went the extra mile and did that. Check if it's okay